README.md

```diff
@@ -37,6 +37,8 @@ This application is the backend for a property mapping application that visualis
 Run the CSV reader script to load data into PostgreSQL: 
 `node src/utils/csvReader.js`
 
+4. Set up a Cron job to run batchGeocode.js every minute until all data rows have been populated with latitudes and longitudes
```

Comment:
cron job and mention needing to set cron job up with env vars for Postgres client setup. Also specify time it will take for data to be completely updated